assets/src/edit-story/components/helpCenter/scheduledTransition.js

```diff
@@ -32,7 +32,7 @@ const machine = {
     entering: { entered },
     entered: { exiting },
     exiting: { exited },
-    exited: {},
+    exited: { entering },
```

Comment:
Since the checklist isn't unmounting we want the exit state to return to entering so that the transitions stay intact. This has no consequences for the help center which also uses this because the helpcenter unmounts.